### PR TITLE
zh-translation: /docs/setup/install/multicluster/simplified/index.md

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1498
ref: https://github.com/servicemesher/istio-official-translation/issues/1466

### CODE REVIEW

File: content/zh/docs/setup/install/multicluster/simplified/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+这将部署 Istio 到每个集群，并配置每个势力以互相安全通信：
+
+{{< text bash >}}
+$ ./setup-mesh apply
+{{< /text >}}
+
+想要往网格中添加或删除集群，只需要对应地更新该拓扑文件并重新应用这些更改。
+
+{{< warning >}}
+每当您使用 `setup-mesh.sh apply` 时，都会在您的工作目录中创建一些秘密材料文件，尤其是与不同证书关联的一些私钥。
```
Comment (vivian99-wu) [inline content/zh/docs/setup/install/multicluster/simplified/index.md]: 秘密材料文件

===》

建议 Secret 不翻译。这样不容易与中文的形容词混淆，更直接指代集群安全所需的secret文件。您说呢？

Comment (ilylia) [reply content/zh/docs/setup/install/multicluster/simplified/index.md]: fixed

**Pull request merged**